frontend/src/components/Spinner/Spinner.tsx

```diff
@@ -5,13 +5,25 @@ import S from './styles';
 
 type Props = {
   text: string;
+  component?: string;
 };
 
-const Spinner = ({ text }: Props) => (
-  <S.Wrapper className="loading-spinner">
-    <Spin indicator={<LoadingOutlined style={{ fontSize: '2.5rem' }} spin />} />
-    <p>{text}</p>
-  </S.Wrapper>
-);
+const Spinner = ({ text, component = '' }: Props) => {
+  // modify spinner css for course tab loads
+  if (component === 'course-tab') {
+    return (
+      <S.Wrapper className="loading-spinner" style={{ gap: '1px' }}>
```

Comment:
and then you can make two wrapper styles for small or large, and have no on-element styling.
(orrrrr rewrite styled wrapper to be an antd Flex as we woed about last meeting :sob:)

Reply:
alsoooooo, make small a flex row